myapp/app/controllers/tasks_controller.rb

```diff
@@ -1,10 +1,17 @@
 # frozen_string_literal: true
 
 class TasksController < ::ApplicationController
+  DEFAULT_SORT = 'id ASC'
+  ORDER_MAPPING = {
```

Comment:
We have to add both `ASC` and `DESC` for keys in this case.
So, may be separating with `sort_column` and `order_type` is better. ←It's just a sample

``` .rb
SORT_COLUMN = ['id', 'created_at']
ORDER_TYPE = ['asc', 'desc']
```